app/src/main/res/values-de/strings.xml

```diff
@@ -44,6 +44,14 @@
     <string name="dark">Dunkel</string>
     <string name="x_deleted">%1$s gelöscht</string>
     <string name="undo">Abbrechen</string>
+	<string name ="normal">Normal</string>
+    <string name ="bold">Fett</string>
+    <string name ="extra_bold">Extrafett</string>
+    <string name ="ui_settings">Benutzeroberfläche</string>
+    <string name ="top_bar_font_weight">Schriftstärke in der oberen Leiste</string>
+    <string name ="behaviour">Verhalten</string>
+    <string name ="appearance">Aussehen</string>
+
```

Comment:
Please also remove this empty line, leaving only one empty line between the two lists.

Reply:
fixed.
The second empty line is also present in strings.xml for english language